docs/builds/guides/frameworks/react.md

```diff
@@ -67,18 +67,73 @@ The `<CKEditor>` component supports the following properties:
 * `onChange` &ndash; A function called when the editor data has changed. See the {@link module:engine/model/document~Document#event:change:data `editor.model.document#change:data`} event.
 * `onBlur` &ndash; A function called when the editor was blurred. See the {@link module:engine/view/document~Document#event:blur `editor.editing.view.document#blur`} event.
 * `onFocus` &ndash; A function called when the editor was focused. See the {@link module:engine/view/document~Document#event:focus `editor.editing.view.document#focus`} event.
-* `onError` &ndash; A function called when the editor has crashed during the initialization. It receives the error object as a parameter.
+* `onError` &ndash; A function called when the editor has crashed during the initialization or during the runtime. It receives the error instance and the error details (`phase` and `willEditorRestart`) as arguments.
 
-The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two parameters:
+The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two arguments:
 
 1. An {@link module:utils/eventinfo~EventInfo `EventInfo`} object.
 2. An {@link module:core/editor/editor~Editor `Editor`} instance.
 
+### Context feature
+
+The `ckeditor5-react` integration comes with a ready-to-use component for the {@link features/context-and-collaboration-features Context feature} that is useful when used together with some {@link features/collaboration CKEditor 5 collaboration features}.
+
+```jsx
+import React, { Component } from 'react';
+import { CKEditor, CKEditorContext } from '@ckeditor/ckeditor5-react';
+
+// Assuming that application builds the editor from source.
+import Context from '@ckeditor/ckeditor5-core/src/context';
+import ClassicEditor from './ckeditor';
+
+class App extends Component {
+	render() {
+		return (
+			<div className="App">
+				<CKEditorContext context={ Context }>
+					<h2>Using CKeditor 5 Context feature in React</h2>
+					<CKEditor
+						editor={ ClassicEditor }
+						data="<p>Hello from the first editor working with the context!</p>"
+						onReady={ editor => {
+							// You can store the "editor" and use when it is needed.
+							console.log( 'Editor1 is ready to use!', editor );
+						} }
+					/>
+
+					<CKEditor
+						editor={ ClassicEditor }
+						data="<p>Hello from the second editor working with the context!</p>"
+						onReady={ editor => {
+							// You can store the "editor" and use when it is needed.
+							console.log( 'Editor2 is ready to use!', editor );
+						} }
+					/>
+			</div>
+		);
+	}
+}
+```
+
+The `CKEditorContext` component supports the following properties:
+
+* `config` &ndash; The CKEditor 5 context configuration.
+* `context` &ndash; {@link module:core/context~Context The CKEditor 5 context class}.
+* `isLayoutReady` &ndash; A property that delays the editor creation when set to true. Useful when the CKEditor 5 annotations or a presence list is used.
```

Comment:
In other words – the property waits with the editor's initialization for other elements that must be available in the [DOM](https://developer.mozilla.org/en-US/docs/Glossary/DOM).

A question – how should I inform the component that the element (that we wait for) is ready?

Reply:
`isLayoutReady={true}`. Or maybe I misunderstood you?

Reply:
What does the option do?

Reply:
Oh, now I have read my comment again :facepalm: